.github/actions/common/mq-grayscale/rocketmq/action.yml

```diff
@@ -0,0 +1,59 @@
+name: "Rocketmq Message Gray Common operations"
+description: "Do something common for rocketmq message gray"
+runs:
+  using: "composite"
+  steps:
+    - name: Set up JDK ${{ env.javaVersion }}
+      uses: actions/setup-java@v3
```

Comment:
use v4 version for setup-java、cache、checkout action

Reply:
fixed